cocos/tween/tween.ts

```diff
@@ -84,11 +86,13 @@ export class Tween<T> {
      * @param other @en The rear tween of this tween @zh 当前缓动的后置缓动
      */
     then (other: Tween<T>): Tween<T> {
-        if (other instanceof Action) {
-            this._actions.push(other.clone());
-        } else {
-            this._actions.push(other._union());
-        }
```

Comment:
`then` only accesses `Tween<T>`, if passing an `Action`, I added another private method `insertAction` for that.